testsuite/lb_bulk_stress.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import print_function
+
+import espressomd as md
+from espressomd import lb 
+from espressomd import observables
+
+import numpy as np
+import unittest as ut 
+
+box_l = 9
+sys = md.System(box_l=[box_l, box_l, box_l])
+sys.time_step = 0.01
+sys.cell_system.skin = 0.4
+
+@ut.skipIf(not md.has_features(["LB"]),
+          "Features not available, skipping test!")
+class LBTest(ut.TestCase):
+
```

Comment:
Please remove those blank lines and add a short docstring which describes what the test is doing.